docs/integrations/sources/mssql.md

```diff
@@ -341,7 +341,8 @@ WHERE actor_definition_id ='b5ea17b1-f170-46dc-bc31-cc744ca984c1' AND (configura
 ## Changelog
 
 | Version | Date       | Pull Request                                                                                                      | Subject                                                                                                                                         |
-|:--------|:-----------|:------------------------------------------------------------------------------------------------------------------| :---------------------------------------------------------------------------------------------------------------------------------------------- |
+| :------ | :--------- | :---------------------------------------------------------------------------------------------------------------- | :---------------------------------------------------------------------------------------------------------------------------------------------- |
+| 2.0.2   | 2023-10-25 | [31834](https://github.com/airbytehq/airbyte/pull/31834)                                                          | Add explicit support for Always On read-only replicas                                                                                           |
```

Comment:
Is this version going to be release? If so I ask to update our documentation above explaining the usage of this feature.